LEAF_Request_Portal/form.php

```diff
@@ -1138,7 +1138,8 @@ public function doModify($recordID)
         {
             // If form has _selected key use over initial key (Multi-Select Dropdown)
             if (is_numeric($key) && $_POST[$key . '_selected']) {
-                $_POST[$key] = $_POST[$key . '_selected'];
+                $strval = $_POST[$key . '_selected'];
```

Comment:
Are the $_POST variables getting sanitized somewhere?

Reply:
I made some updates to sanitize the string that contains the multiselect options.  Also noted the possible issue with checkboxes format (checkboxes format is not related to this ticket)